layouts/partials/head.html

```diff
@@ -75,6 +75,10 @@
     {{ $image = print .Params.header_image_url }}{{ end }}
 
 
+  {{ if eq .Params.layout "faq-question" }}
+    {{ $description = print .Params.summary }}{{ end }}
+
+
```

Comment:
will approve if you remove extra new lines

Reply:
updated the code change to reflect the suggestion!